f/fluent-hc-4.1.x/fluent-hc-4.1.x_ubuntu_16.04.sh

```diff
@@ -0,0 +1,11 @@
+# Install dependencies
+apt update -y
+apt install git maven openjdk-8-jdk
+
+git clone https://github.com/dthadi3/fluent-hc-4.1.x.git
+
+cd fluent-hc-4.1.x/
```

Comment:
Is it because there are no branches, we are not using any command line argument ?

Reply:
yes